pages/common/terragrunt.md

```diff
@@ -0,0 +1,28 @@
+# terragrunt
+
+> Keep your Terraform CLI arguments DRY.
+> More information: <https://terragrunt.gruntwork.io/>.
+
+- Generate and show an execution plan:
+
+`terragrunt plan`
+
+- Build or change infrastructure:
+
+`terragrunt apply`
+
+- Show current deployment (from state):
+
+`terragrunt show`
+
+- Show module output values:
+
+`terragrunt output`
+
+- Destroy Terraform-managed infrastructure:
+
+`terragrunt destroy`
+
+- Build or change infrastructure from a tree of terragrunt modules (stack):
```

Comment:
what do stack means?
```suggestion
- Build or change infrastructure from a tree of Terragrunt modules (stack):
```

Reply:
'Stack' is used to refer to a tree of Terragrunt modules.